volestipy/README.md

```diff
@@ -1,28 +1,70 @@
+Through the [Download center](https://www.gurobi.com/downloads/) you may download its Python interface along with a license; the latter is a text file called `gurobi.lic`.
+Without a license *gurobipy* will not be able to perform neither will *volestipy*. 
+
+You can follow the inscriptions described [here](https://support.gurobi.com/hc/en-us/articles/360044290292-Installing-Gurobi-for-Python) to get ```gurobipy```. 
+
+Once you complete these steps, make sure that `gurobipy` is now available for your Python environment. 
+
+```
+haris@XPS-13-9343:~$ python3
+Python 3.6.9 (default, Apr 18 2020, 01:56:04) 
+[GCC 8.4.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import gurobipy
+>>> gurobipy.
+gurobipy.AttrConstClass(     gurobipy.GenExpr(            gurobipy.QConstr(            gurobipy.atexit              gurobipy.gurobi(             gurobipy.multidict(          gurobipy.resetParams(
+gurobipy.Batch(              gurobipy.GurobiError(        gurobipy.QuadExpr(           gurobipy.bi                  gurobipy.help(               gurobipy.numbers             gurobipy.setParam(
+gurobipy.CallbackClass(      gurobipy.Iterable(           gurobipy.SOS(                gurobipy.dis                 gurobipy.inspect             gurobipy.operator            gurobipy.sys
+gurobipy.Column(             gurobipy.LinExpr(            gurobipy.StatusConstClass(   gurobipy.disposeDefaultEnv(  gurobipy.itertools           gurobipy.or_(                gurobipy.system(
+gurobipy.Constr(             gurobipy.MLinExpr(           gurobipy.TempConstr(         gurobipy.exprfactory(        gurobipy.izip(               gurobipy.os                  gurobipy.tupledict(
+gurobipy.Env(                gurobipy.MQuadExpr(          gurobipy.Var(                gurobipy.exprfactory_iter(   gurobipy.logging             gurobipy.paramHelp(          gurobipy.tuplelist(
+gurobipy.ErrorConstClass(    gurobipy.MVar(               gurobipy.abs_(               gurobipy.fnmatch             gurobipy.math                gurobipy.quicksum(           gurobipy.types
+gurobipy.GRB(                gurobipy.Model(              gurobipy.all_(               gurobipy.gc                  gurobipy.max_(               gurobipy.re                  gurobipy.writeParams(
+gurobipy.GRBStringIO(        gurobipy.ParamClass(         gurobipy.and_(               gurobipy.getParamInfo(       gurobipy.min_(               gurobipy.read(               
+gurobipy.GenConstr(          gurobipy.ParamConstClass(    gurobipy.any_(               gurobipy.glob                gurobipy.models(             gurobipy.readParams(         
+>>> gurobipy.
+```
+
+
+### Install *volestipy*
+
+After getting the dependencies, download the *volesti* repository from GitHub and run the following commands.
 
 ```
 cd volestipy
-LDFLAGS="-L/usr/lib/lpsolve/" python setup.py build_ext -i
-python setup.py install -f
+LDFLAGS="-L/usr/lib/lp_solve/" python3 setup.py install --user
```

Comment:
you both use `lp_solve` and 'lpsolve` in README, please be consistent

Reply:
replcaced ```lp_solve``` with ```lpsolve```